### PR TITLE
document what exists of our container publishing strategy

### PR DESCRIPTION
**Pull request by no-reply**
adds some documentation about publishing containers to https://hub.docker.com/u/samveralabs

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by jeremyf** [comment]
I'm going to step through the documentation to verify.

File: CONTAINERS.md [hunk 1]
```diff
@@ -35,17 +35,33 @@ It also runs database migrations.
 We also provide a base image which can be reused for Hyrax applications: `hyrax`.
 
 ```sh
-echo "FROM hyrax" > Dockerfile
+echo "FROM samveralabs/hyrax" > Dockerfile
```
Comment (jeremyf) [inline CONTAINERS.md:38]: As someone Docker naive, from where would I run this? I don't think I'd want to run it in the root of my hyrax git repository.

File: CONTAINERS.md [hunk 2]
```diff
@@ -35,17 +35,33 @@ It also runs database migrations.
...
 
-At the moment, we don't publish hyrax images anywhere (TODO). If you want to use them, you need to build them yourself:
+We publish several Hyrax images to hub.docker.com under the
+[`samveralabs` group][dockerhub-samveralabs]. To build them, do:
 
 ```sh
 # build an image for an app using Postgresql (`gem 'pg'`)
-docker build --target hyrax --tag hyrax .
+docker build --target hyrax --tag samveralabs/hyrax:(git rev-parse HEAD) .
+docker push samveralabs/hyrax:(git rev-parse HEAD)
```
Comment (jeremyf) [inline CONTAINERS.md:49]: I'm assuming the `docker push` command would push up the image to `samveralabs`.  Is that correct?  Should we document that? What about how to get permission to do that?

File: CONTAINERS.md (hunk 1 above)
Comment (no-reply) [reply CONTAINERS.md:38]: in the root of your own hyrax application.

maybe the context of this isn't clear?

File: CONTAINERS.md (hunk 2 above)
Comment (no-reply) [reply CONTAINERS.md:49]: that's right. this is to publish the image.

right now, there's no strategy for handing out permissions. i had to bang the drum to get them myself.

File: CONTAINERS.md (hunk 1 above)
Comment (jeremyf) [reply CONTAINERS.md:38]: It's not immediately clear, but it's not inscrutable.

File: CONTAINERS.md [hunk 3]
```diff
@@ -32,20 +32,38 @@ It also runs database migrations.
...
 
-We also provide a base image which can be reused for Hyrax applications: `hyrax`.
+We also provide a base image which can be reused for your Hyrax applications: `hyrax`.
 
 ```sh
-echo "FROM hyrax" > Dockerfile
+echo "FROM samveralabs/hyrax" > Dockerfile
 ```
 
+_This is for applications that mount Hyrax and is separate from the docker containers for Hyrax engine development._
```
Comment (no-reply) [inline CONTAINERS.md:41]: one of the things i hope this documentation will eventually capture is that these containers are separate, in that we have to take action to maintain them separately; but also that they are closely related. the two images use the same base, and (hopefully) the `hyrax-engine-dev` is the same as the `hyrax` image in every substantial way.

**Pull request merged**